aws-predictions/src/main/java/com/amplifyframework/predictions/aws/service/RunFaceLivenessSession.kt

```diff
@@ -183,4 +183,17 @@ internal class RunFaceLivenessSession(
         livenessWebSocket.clientStoppedSession = true
         reasonCode?.let { livenessWebSocket.destroy(it) } ?: livenessWebSocket.destroy()
     }
+
+    private fun getStreamingEndpointForRegion(region: String): String {
+        val baseDomain = when {
+            region.startsWith("us-isof", ignoreCase = true) -> ISO_PARTITION_BASE_DOMAIN
+            else -> DEFAULT_BASE_DOMAIN
+        }
+        return "wss://streaming-rekognition.${region}.${baseDomain}:443"
```

Comment:
Please run `./gradlew ktlintCheck` because it's failing on this line. You can rewrite this to:
`return "wss://streaming-rekognition.$region.$baseDomain:443" `

Reply:
Update the code now.  Running `./gradlew ktlintCheck` and build is successful